app/models/product.rb

```diff
@@ -5,26 +5,28 @@
 #  id          :bigint           not null, primary key
 #  deleted_at  :datetime
 #  description :string
+#  enabled     :boolean          default(TRUE), not null
```

Comment:
Could we add presence validation of the enabled attribute in the model and in spec?